test/integration/api/services_controller_test.rb

```diff
@@ -90,6 +90,35 @@ class SettingsTest < self
       assert hosted_option.attribute('checked').present?
     end
 
+    test 'shows authentication options' do
+      # Default value
+      service.stubs(:backend_version).returns(nil)
```

Comment:
```suggestion
      service.stubs(:backend_version)
```

if you return `nil`, no need to specify it